### ISSUE TITLE
Note: Non-descriptive link text

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds.com#15]
For example, from #4: The link text such as “PDF version" and  "Word version” in the main content area does not describe the destination page it points to.
As a result, screen reader users failed to understand the purpose of the links and which page will get activated on accessing the links.

Code Snippet:
> \<a href="/doc/service-provider-quick-reference-guide.pdf">PDF version</a>
> (..)
> \<a href="/doc/service-provider-quick-reference-guide-template.docx">word version</a>

Including but not limited to - please check for this existing elsewhere:

Page Name | Accessibility Report Issue Number
------------|-----------------------------------
Resources | # 4
Communications | # 7
Communications | # 8
Communications | # 16

Numbers are the issue numbers from the Accessibility Report on the Findings worksheet at https://docs.google.com/spreadsheets/d/1fQgaUcbWK5n8gUorLlAoITfGaJC0Gqg1jzuur9wPoZg/edit#gid=585536028

**Comment by jrbenn** [comment]
- [ ] # 4
- [ ] # 7 Not updating. This is what was sent.
- [ ] # 8 Not updating. This is what was sent.
- [ ] # 16 Not updating. This is what was sent.
